questionnaires/covid19_symptoms/covid19_symptoms_armt.json

```diff
@@ -0,0 +1,866 @@
+        "field_label": "Rate how severe your breathing has been",
+        "select_choices_or_calculations": [
+            {
+                "code": "0",
+                "label": "0 None "
+            },
+            {
+                "code": "1",
+                "label": "1 Mild "
+            },
+            {
+                "code": "2",
+                "label": "2 Moderate "
+            },
+            {
+                "code": "3",
+                "label": "3 Severe"
+            }
+        ],
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "[breath_2wk] = '1'",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "",
+        "matrix_ranking": "",
+        "field_annotation": "range-type",
+        "evaluated_logic": "responses['breath_2wk'] ==  '1'"
+    },
+    {
+        "field_name": "fever_wk",
+        "form_name": "covid19_symptoms",
+        "section_header": "",
+        "field_type": "yesno",
+        "field_label": "Have you had a fever (including feeling cold and shivering) in the last two weeks?",
+        "select_choices_or_calculations": "",
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "",
+        "matrix_ranking": "",
+        "field_annotation": "",
+        "evaluated_logic": ""
+    },
+    {
+        "field_name": "fever_rate",
+        "form_name": "covid19_symptoms",
+        "section_header": "",
+        "field_type": "range",
+        "field_label": "Rate how severe your fever has been",
+        "select_choices_or_calculations": [
+            {
+                "code": "0",
+                "label": "0 None "
+            },
+            {
+                "code": "1",
+                "label": "1 Mild "
+            },
+            {
+                "code": "2",
+                "label": "2 Moderate "
+            },
+            {
+                "code": "3",
+                "label": "3 Severe"
+            }
+        ],
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "[fever_wk] = '1'",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "",
+        "matrix_ranking": "",
+        "field_annotation": "range-type",
+        "evaluated_logic": "responses['fever_wk'] ==  '1'"
+    },
+    {
+        "field_name": "breath_2wk",
```

Comment:
@afolarin Should this question be before `Rate how severe your breathing has been` just like order of the 2 fever questions before this?

Reply:
Yes, thought it was